app/pages/angelfish.tsx

```diff
@@ -32,7 +32,7 @@ const Home: React.FC<HomePageProps> = (props: HomePageProps) => {
       </Head>
       <section className="pb-12 pt-12 md:pt-16">
         <div className="page-container mx-5 lg:mx-auto">
-          <div className="flex flex-row justify-between">
+          <div className="flex flex-row justify-content-between">
```

Comment:
Note change in convention from tailwind. I like the tailwind naming, but this is nice too imo